tests/unit/test_passthru_args.py

```diff
@@ -0,0 +1,43 @@
+try:
+    import unittest.mock as mock
+except ImportError:
+    import mock
+
+from .util import DjangoSetupMixin
+
+
+@mock.patch('behave_django.management.commands.behave.behave_main', return_value=0)  # noqa
+@mock.patch('behave_django.management.commands.behave.BehaviorDrivenTestRunner')  # noqa
+class TestPassThruArgs(DjangoSetupMixin):
+
+    def test_keepdb_flag(self,
+                         mock_test_runner,
+                         mock_behave_main):
+        """Test if keepdb is properly set on the test_runner."""
+
+        self.run_management_command('behave', keepdb=True)
+        assert mock_test_runner.call_args[1]['keepdb'] is True
```

Comment:
Should we maybe write that like this? (in all occurrences)
```python
    args, kwargs = mock_test_runner.call_args
    assert kwargs['keepdb'] is True
```
To make it self-explanatory.

Reply:
@alex-hutton Would you care to adapt the code as I proposed?

@mixxorz Any opinion on the whole PR as such? (See my [comment above](https://github.com/behave/behave-django/pull/62#issuecomment-443553325).)